prisma/migrations/20250906181432_group_recruit_status_enum_and_default/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- CreateEnum
+CREATE TYPE "GroupRecruitStatus" AS ENUM ('RECRUITING', 'ALWAYS_OPEN', 'CLOSED');
```

Comment:
```suggestion
CREATE TYPE "GroupRecruitStatus" AS ENUM ('OPEN', 'ALWAYS_OPEN', 'CLOSED');
```
always_open,closed 다 상태여서 open으로 가는 게 맞을 것 같아요. recruiting은 혼자 너무 진행형인듯..

Reply:
넵 closed랑 반의어이기도 하고 always_open이랑 일맥상통할것같아요.